wasmplugin/plugin.go

```diff
@@ -365,3 +405,24 @@ func logError(error ctypes.MatchedRule) {
 		proxywasm.LogDebug(msg)
 	}
 }
+
+type AddressInfo struct {
+	IP   string
+	port int
+}
+
+func parseAddress(rawAddress []byte) (AddressInfo, error) {
+	// Split address and port
+	splittedAddr := regexp.MustCompile(`(.*):([0-9]+)$`).FindStringSubmatch(string(rawAddress))
+	if splittedAddr == nil {
+		return AddressInfo{
```

Comment:
I set default values considering not a fatal error if something goes wrong on retrieving these values. That's how it's worked up to now. Can it work?